open-machine-learning-jupyter-book/data-science/working-with-data/pandas2.ipynb

```diff
@@ -0,0 +1,2374 @@
+{
```

Comment:
<p>Optionally, ## can be used as a place of separation rather than ###</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>